src/Avatar/Avatar.js

```diff
@@ -18,7 +18,9 @@ const Avatar = ({
   const _modifier = modifier ? `avatar--${modifier}` : '';
   const ref = useRef(null);
 
-  useEffect(() => setSize(ref.current.clientHeight / 2));
+  useEffect(() => {
+    if (ref.current) setSize(ref.current.offsetHeight / 2);
```

Comment:
In mi opinion we shouldn't have this here. We have coupled the styles in the js files. To me would be better have like a standard sizes by classnames and switch between classnames through a modifier.